docs/INSTALL.md

```diff
@@ -0,0 +1,63 @@
+# Installation with `pyenv` and `pdm`
+`pyenv` allows you to install multiple python versions side by side in your local user environment. This installation method is preferred over using `conda` to install `pdm`
+
+## Install pyenv
+1. Install pyenv: Follow the `pyenv` installation instructions [here](https://github.com/pyenv/pyenv#installation). There are just two steps.
+1.  Download and install `pyenv`
+2. Add `pyenv` functions to your shell configuration.
+
+## Install python 3.9
+Navigate to the `cellPACK` folder and install the required python version.
+```bash
+pyenv install $(cat .python-version)
+```
+The `.python-version` file in this repo tells `pyenv` to load Python 3.9 when you are inside the `cellPACK` directory.
+Check that this is working with `which python && python --version`.
+
+## Install `pdm`
+Detailed installation instructions are available [here](https://pdm.fming.dev/latest/#installation).
+For Linux or MacOS, install `pdm` for your user as follows.
+
+1. Download the installer
+
+```bash
+curl -sSLO https://pdm.fming.dev/install-pdm.py
```

Comment:
In my case, this command didn't trigger the installation, I used the one from the instruction (`curl -sSL https://pdm-project.org/install-pdm.py | python3 -`) instead, and it worked smoothly.

Reply:
This command just downloads the installer. Step 3 in the instructions (`python install-pdm.py`) runs the downloaded installer and installs pdm. Thanks for checking this!